shared/pub/metadata.js

```diff
@@ -49,3 +49,49 @@ export const getTextAbstract = (docJson) => {
 	}
 	return abstract;
 };
+
+export const getGSNotes = (notes) => {
+	return notes.reduce((unique, note) => {
+		const noteArray = [];
+		const noteType = note.json[0].type;
+		const noteTypes = {
+			chapter: 'book',
+			book: 'book',
+			proceedings: 'conference',
+			'paper-conference': 'conference',
+		};
+		const noteTypeString = noteTypes[noteType] || 'journal';
+
+		const noteArr = Object.entries(note.json[0]);
+		noteArr.forEach((entry) => {
```

Comment:
Personally I'd find this a bit easier to read as `Object.entries(...).forEach(([key, value]) => { ... })`. Something about `entry[0]` or `entry[1]` just adds cognitive overhead in remembering what it is I'm looking at. And having `noteArray` and `noteArr` in the same scope is a little confusing too.

Reply:
Done, thanks. Had no idea you could do that in forEach (is that...destructuring, what is that?). Anyhow, super helpful.